src/renderer/component/inspect-request/request-details.js

```diff
@@ -49,10 +49,18 @@ export default class RequestDetails extends React.Component {
         Request Headers:
         <RequestMetadata metadata={request.request.headers} />
       </section>
+      <section>
+        Request Body:
+        <RequestMetadata metadata={request.request.body} />
```

Comment:
Should this be `request.request.body.string`? Isn't `request.request.body` going to be:
```
{
    string: '<body>'
}
```
?

Reply:
As far I could see metadata need to be an object (Dict). It would probably better to have a specialised component to display body. I don't have the expertise to create one (These were my first lines of code around a react project, and I am aware I did not do any react coding here).

Reply:
Oh, shoot, you're right. It looks like `<RequestMetadata>` component is meant to display an object of data, yeah.
Perhaps we can get away with making this section:
```
<section>
  Request Body:
  <RequestMetadata metadata={request.request.body} />
</section>
```